mobile-save-for-later/conf/cfn.yaml

```diff
@@ -233,3 +236,35 @@ Resources:
            Path: "/syncedPrefs/me"
            Method: GET
            RestApiId: !Ref SaveForLaterApi
+
+  SaveForLaterReadThrottleEvents:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      Namespace: AWS/DynamoDB
+      MetricName: ReadThrottleEvents
+      Unit: Count
+      Statistic: Sum
+      ComparisonOperator: GreaterThanOrEqualToThreshold
```

Comment:
If the threshold is 0, should this be `GreaterThanThreshold`?

Reply:
yep, sorry

Reply:
No need to be sorry, I didn't notice it until I'd approved the PR either! 😅